po/en.po

```diff
@@ -1778,8 +1778,10 @@ msgstr "Illegal file descriptor in redirection “%ls”"
 
 #: parse_constants.h:213
 #, c-format
-msgid "No matches for wildcard '%ls'."
-msgstr "No matches for wildcard “%ls”."
+msgid "No matches for wildcard pattern '%ls'.\n"
+"(Tip: To avoid wildcarding, quote it. To allow no matches, expand it to a variable.)"
```

Comment:
What's the concern?

Replacing "expand" with "assign" may be more beginner friendly, but at the cost of preciseness: It might also be understood as quoting the glob when assigning the variable, hoping the glob will instead be interpreted when the variable is expanded (which we thankfully don't support, unlike bash).

I've opted to call it "wildcard pattern" instead of "glob". Isn't that what we call it in the more user-facing documentation?